src/store/modules/product/actions.ts

```diff
@@ -32,7 +32,6 @@ const actions: ActionTree<ProductState, RootState> = {
         const totalProductsCount = resp.data.response.numFound;
 
         if (payload.viewIndex && payload.viewIndex > 0) products = state.products.list.concat(products)
-        commit(types.PRODUCT_SEARCH_UPDATED, { products: products, totalProductsCount: totalProductsCount })
```

Comment:
If find product action is not used anywhere, we could remove it

Reply:
Removed `findProduct` action from the product module.